src/parser/deathknight/frost/modules/features/BreathOfSindragosa.tsx

```diff
@@ -19,8 +19,8 @@ class BreathOfSindragosa extends Analyzer {
   totalDuration = 0;
   breathActive = false;
 
-  constructor(...args) {
-    super(...args);
+  constructor(args: any) {
+    super(args);
     this.active = this.selectedCombatant.hasTalent(SPELLS.BREATH_OF_SINDRAGOSA_TALENT.id);
```

Comment:
args should be options, it probably works here, but you can't add abilities if you wanted to for example.